### ISSUE TITLE
Charlie station circuit printer unusable due to tax.

### ISSUE THREAD

**Issue by TheDrunkenPumpkin** [tgstation/tgstation#66690]
Reporting client version: 514.1583

## Round ID:
182587

## Testmerges:
- [Parallax but better: Smooth movement cleanup](https://www.github.com/tgstation/tgstation/pull/66567)
- [Dynamic 2022, Part 1 - Redesigning midround rolls (Midrounds happen far more frequently now, and scale their power over time)](https://www.github.com/tgstation/tgstation/pull/65997)
- [Keep mutating filenames when the cdn is disabled midround.](https://www.github.com/tgstation/tgstation/pull/66550)
- [Makes smoke and foam attempt to fill the available space.](https://www.github.com/tgstation/tgstation/pull/65281)

**Comment by Maurukas** [comment]
A thought:

It would make more sense to have the tax be applied to taking materials from the ore silo. Only station lathes are linked to silo round start, and additional ones can only be linked through player action.

Any solution that relies on subtypes, variables, or z-level traits is going to run into an endless series of what-ifs and edge cases, like having a golem ship on the same Z as the station, or deconstructing the lathe losing the trait. Or any other contrived situation a mapper comes up with.

**Comment by moocowswag** [comment]
> It would make more sense to have the tax be applied to taking materials from the ore silo. Only station lathes are linked to silo round start, and additional ones can only be linked through player action.

Good idea, I hate it when I built my own lathe/printer and supply it with my own materials just for it to charge me credits.

**Comment by ArcaneMusic** [comment]
This appears to have been fixed by 66566.
